src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -441,6 +447,13 @@ public boolean isOpen() {
     @Override
     public void close() throws IOException {
         if (open) {
+            exec.shutdown();
+            try {
+                exec.awaitTermination(3, TimeUnit.MINUTES);
```

Comment:
It's fine to call `shutdownNow` right away - no one's going to read the data being prefetched so there's no point in waiting 10s before interrupting the prefetching. Instead, let's interrupt at once and give the thread 10s to notice and stop cleanly.

I tried testing this, but the reads were too fast so I added a sleep in there. The output shows that the prefetching read gets interrupted as we expect (so we don't wait the full 10s), and the caller of close() gets no exception. 

> WorkUnit.call() starting
> WorkUnit.call() starting for real now
> WorkUnit.call() done
> isInterrupted: false
> WorkUnit.call() starting
> shutdownNow
> sleep was interrupted
> WorkUnit.call() starting for real now
> WorkUnit.call interrupted by: java.nio.channels.ClosedByInterruptException
> close
> Call to close() took: 0ms.

That said, the same test also shows that just closing the channel's fine too in the sense that the caller of close() sees no exception, so I guess this wasn't really necessary. I'll leave in the awaitTermination but cut the timeout to a single second since there's no penalty for timing out.